src/app/shared/styles/toggle-button.css

```
@@ -0,0 +1,35 @@
+.mat-button-toggle-checked {
+  background-color: rgba(56, 73, 249, 0.07);
+  color: #3849f9;
+  border-top-left-radius: 1.2rem;
+  border-bottom-left-radius: 1.2rem;
+  border: 2px solid #3849f9;
+}
+
+.mat-button-toggle-group {
+  background-color: none !important;
+  align-items: center;
+  background: #ffffff;
```

Biome (1.9.4)

[error] Unexpected shorthand property background after background-color



(lint/suspicious/noShorthandPropertyOverrides)